### ISSUE TITLE
booting no passe "3 connecting"

### ISSUE THREAD

**Issue by abdelatif777** [GaryOderNichts/udpih_nxpayload#1]
Hello, thanx for this exploit.
I trying to debrick a wiiu, with a switch,but the processus stuck at the third tentative of connections.
What i can do?

**Comment by abdelatif777** [comment]
hello, my attempt from a switch to launch the mode stops at step 3 maximum.should i use files in the sd card of the wiiu?
* tiramisu exploit

**Comment by GaryOderNichts** [comment]
Did you make sure to check all of this?

> :warning: Important notes for this to work:
> - Make sure **no** other USB Devices are attached to the console.
> - Only use USB ports on the front of the console, the back ports **will not** work.
> - If your console has standby mode enabled, pull the power plug and turn it on from a full coldboot state.

**Comment by abdelatif777** [comment]
Thanks for the reply, yes I have checked all these points.using an unpatched nintendo switch, do i also have to with the recovery menu in the root of the wiiu sd card?

**Comment by abdelatif777** [comment]
Hi, I managed to launch the recovery mode, I installed the title with "success", but the console still does not pass the "wiiu" screen. It is possible to connect with the wupserver, can it be used for something or is it a waste of time while waiting for another solution? I have the logs, can that direct me to the cause of the failure? I have no dump of my nand, I don't know if we can modify the nand with wupserver? Thanks

**Comment by abdelatif777** [comment]
Hi, I managed to launch the recovery mode, I installed the title with "success", but the console still does not pass the "wiiu" screen. It is possible to connect with the wupserver, can it be used for something or is it a waste of time while waiting for another solution? I have the logs, can that direct me to the cause of the failure? I have no dump of my nand, I don't know if we can modify the nand with wupserver? Thanks
